### PR TITLE
Dotnet new templates doc

### PR DESCRIPTION
**Pull request by Skrypt**
https://github.com/OrchardCMS/OrchardCore/tree/dotnetNewTemplates-doc/src/Templates

### CODE REVIEW

File: src/Templates/README.MD [hunk 1]
```diff
@@ -0,0 +1,211 @@
+# Orchard Core Templates
+
+Orchard Core Templates uses dotnet new templates configurations for creating new themes and modules from command shell. Our configurations have been implemented by using examples that you can find there : https://github.com/dotnet/dotnet-template-samples
+
+## Create a new website
+
+### From Visual Studio
```
[the remark this answers is not in the document]
Comment (Skrypt) [reply src/Templates/README.MD]: It shows how to do it both ways. "Manually" or with ­`dotnet new` Templates. It's not bad to display both. Though, the VS doc should be also part of "Getting Started" doc. If we remove it here then let's at least move it to a different doc before.

File: src/Templates/README.MD [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+## Create a new module
+
+### From Visual Studio
+
+Fire up Visual Studio, open Orchard Core solution file (.sln), select OrchardCore.Modules folder, right click and select "add --> new project" and create a new .NET Standard Class Library:
+
+![image](/src/docs/assets/images/38450533-6c0fbc98-39ed-11e8-91a5-d26a1105b91a.png)
+
+We strongly suggest that you name your modules like this : **MyModule.OrchardCore** if you consider sharing this module through Nuget afterward ; Since **OrchardCore.Something** is already taken by Orchard Core Nuget feed. Alternatively if you want to name your modules **OrchardCore.Something** you could use Paket https://fsprojects.github.io/Paket/ for referencing those as Github repositories or simply pointing your Nuget packages to a local folder.
```
[the remark this answers is not in the document]
Comment (Skrypt) [reply src/Templates/README.MD]: The modules names is a suggestion that you made in a previous meeting. I removed those lines.

File: src/Templates/README.MD [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+## Adding Orchard Core Nuget Feeds
+
+![image](/src/docs/assets/images/38450194-c617148a-39e7-11e8-95b0-2d35f43a6fad.png)
+
+![image](/src/docs/assets/images/38450242-886933f6-39e8-11e8-896c-1f807e5530a0.png)
+
+#### MyGet:
+
+Dev: https://www.myget.org/gallery/orchardcore-dev
```
[the remark this answers is not in the document]
Comment (Skrypt) [reply src/Templates/README.MD]: removed the dev feed

File: src/Templates/README.MD (hunk 1 above)
[the remark this answers is not in the document]
Comment (Skrypt) [reply src/Templates/README.MD]: I'd prefer keeping it. Somehow it shows what the Templates do automatically. Else the documentation about Templates is somehow from https://github.com/dotnet/dotnet-template-samples

**Comment by sebastienros** [comment]
`REM` is lame. Use a standard text to describe, or create a table with the description as one column.

**Pull request merged**